internal/generator/vector/output/syslog/syslog_test.go

```diff
@@ -112,8 +112,8 @@ var _ = Describe("vector syslog clf output", func() {
 			spec.Syslog = &obs.Syslog{
 				URL:        "tls://logserver:6514",
 				RFC:        obs.SyslogRFC5424,
-				Facility:   "$$.message.facility",
-				Severity:   "$$.message.severity",
+				Facility:   `{.facility||"none"}`,
```

Comment:
i think "none" is not correct fallback value

Reply:
I don't think it matters for the context of the test